common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -370,32 +447,29 @@ public void run() {
         return true;
     }
 
-    private boolean processInvalidUrl(@NonNull final WebView view, @NonNull final String url) {
-        final String lowerCaseUrl = url.toLowerCase(Locale.US);
-        if (isBrokerRequest(getActivity().getIntent())
-                && url.startsWith(AuthenticationConstants.Broker.REDIRECT_PREFIX)) {
-            Logger.error(TAG, "The RedirectUri is not as expected.", null);
-            Logger.errorPII(TAG, String.format("Received %s and expected %s", url, mRedirectUrl), null);
-            returnError(ErrorStrings.DEVELOPER_REDIRECTURI_INVALID,
-                    String.format("The RedirectUri is not as expected. Received %s and expected %s", url,
-                            mRedirectUrl));
-            view.stopLoading();
-            return true;
-        }
+    private boolean processRedirectUriCheck(@NonNull final WebView view, @NonNull final String url) {
+        final String methodName = "#processRedirectUriCheck";
 
-        if ("about:blank".equals(lowerCaseUrl)) {
-            Logger.verbose(TAG, "It is an blank page request");
-            return true;
-        }
+        Logger.error(TAG + methodName, "The RedirectUri is not as expected.", null);
+        Logger.errorPII(TAG, String.format("Received %s and expected %s", url, mRedirectUrl), null);
+        returnError(ErrorStrings.DEVELOPER_REDIRECTURI_INVALID,
+                String.format("The RedirectUri is not as expected. Received %s and expected %s", url,
+                        mRedirectUrl));
+        view.stopLoading();
+        return true;
+    }
 
-        if (!lowerCaseUrl.startsWith(AuthenticationConstants.Broker.REDIRECT_SSL_PREFIX)) {
-            final String redactedUrl = removeQueryParametersOrRedact(url);
+    private boolean processSSLProtectionCheck(@NonNull final WebView view, @NonNull final String url) {
```

Comment:
This method only returns one thing.  A lot of these do.  we could pull their return statements into their callers and have them be void.